podcasts/SwiftUI/NavigationContainer.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+struct NavigationContainer<Content: View>: View {
```

Comment:
This is nice, it's so easy to forget to use `NavigationStack` instead of `NavigationView`. Is it something you plan to apply wider in the codebase?

Reply:
Yes, we have been seen situations that we want to use the newer NavigationStack but been holding up because of our min. IOS requirement, I saw this is an opportunity to finally starting migrating away.

Reply:
I also needed it because of some animation bugs on iOS 18 with NavigationView.